test/packages/deploy_custom_agent_multi_services/_dev/deploy/agent/custom-agent.yml

```diff
@@ -0,0 +1,8 @@
+version: "2.3"
+services:
+  docker-custom-agent:
+    host: pid
+    user: root
+  other-container:
```

Comment:
Would you like to include an example with `build` instructions? I remember you needed it in some of your packages.

Reply:
It looks similar in the package what i am working on


```
version: '2.3'
services:
  docker-custom-agent:
    build: https://raw.githubusercontent.com/agithomas/integrations/issue-4405/packages/oracle/data_stream/memory/_dev/deploy/agent/Dockerfile
    image: elastic-agent-oracle-client
    healthcheck:
      test: ["CMD", "bash", "-c", "echo 'select 1 from dual;' | ORACLE_HOME=/opt/oracle/instantclient_21_4 /opt/oracle/instantclient_21_4/sqlplus -s sys/Oradoc_db1@oracle:1521/ORCLCDB.localdomain?sysdba=1"]
      interval: 10s
      timeout: 10s
      retries: 60
    user: root
  oracle:
    image: docker.elastic.co/observability-ci/database-enterprise:12.2.0.1
    ports:
      - 1521:1521
      - 5500:5500
```

Earlier it was suggested to use as below. 



```
services:
  docker-custom-agent:
    build:
      context: .
    user: root
```

But, this expects Dockerfile to be available in the package-spec repo to avoid the error: 

```

unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/agikthomas/.elastic-package/deployer/docker_custom_agent/Dockerfile: no such file or directory
ERROR: Service 'docker-custom-agent' failed to build : Build failed
```

Reply:
> Can you advice on the path where i can hold this in package-spec repo?

Sorry, I missed this question, please, add it to `spec/integration/_dev/deploy/agent/spec.yml`.

Reply:
Or directly set `additionalContents: false` on this file. This would be consistent with the docker compose runner, see `spec/integration/_dev/deploy/docker/spec.yml`.

Reply:
Added. 

Test Result
```

agikthomas@service-integration-dev-idc-01:~/go/src/github.com/elastic/package-spec$ make test | grep 'multi'
# -count=1 is included to invalidate the test cache. This way, if you run "make test" multiple times
=== RUN   TestPatchedSpec/spec_with_multiple_versions_1.0.0
=== RUN   TestPatchedSpec/spec_with_multiple_versions_2.0.0
    --- PASS: TestPatchedSpec/spec_with_multiple_versions_1.0.0 (0.00s)
    --- PASS: TestPatchedSpec/spec_with_multiple_versions_2.0.0 (0.00s)
=== RUN   TestValidateFile/deploy_custom_agent_multi_services
    --- PASS: TestValidateFile/deploy_custom_agent_multi_services (0.04s)
```